### ISSUE TITLE
Composer error on new installation

### ISSUE THREAD

**Issue by Sammyskills** [lonnieezell/codeigniter-shield#62]
Hi @lonnieezell, 

I just tried installing this package via composer in a fresh CI4 installation (v4.1.9) and it returned the error below:
`Could not find a version of package lonnieezell/codeigniter-shield matching your minimum-stability (stable). Require it with an explicit version constraint allowing its desired stability.`

So, I thought maybe because I haven't installed codeigniter4/settings package. I've installed the package, but the problem still persists. 

What could be the problem ?

**Comment by kenjis** [comment]
Try:
```
composer require lonnieezell/codeigniter-shield:dev-develop
```

Your minimum-stability defined in your composer.json is `stable`, but no stable version of shield exists yet.
So you can't install without an explicit version constraint allowing its desired stabilit.

**Comment by Sammyskills** [comment]
Thanks for the reply. 

I just found out that `stable` is the default value for minimum-stability if not stated in the root composer.json file according to the docs in [composer schema](https://getcomposer.org/doc/04-schema.md#minimum-stability) and in this case for a fresh install of CI v4.1.9, it wasn't stated.

A bit of search on the internet pointed me to the right direction and I used exactly what you recommended.

Thanks once again.
